aggregator/src/cache.rs

```diff
@@ -125,9 +128,25 @@ impl GlobalHpkeKeypairCache {
         let keypairs = self.keypairs.lock().unwrap();
         keypairs.get(id).cloned()
     }
+
+    #[cfg(feature = "test-util")]
+    pub async fn refresh(&self) -> Result<(), Error> {
```

Comment:
I think we should extract the common body of this and the async block in `new()` to a new private method.